src/api/FileActivities.js

```diff
@@ -5,18 +5,23 @@
  */
 
 import Base from './Base';
-import { PERMISSION_CAN_COMMENT, PERMISSION_CAN_VIEW_ANNOTATIONS, ERROR_CODE_FETCH_ACTIVITY } from '../constants';
+import { PERMISSION_CAN_COMMENT, ERROR_CODE_FETCH_ACTIVITY } from '../constants';
 import type { BoxItemPermission } from '../common/types/core';
 import type { ElementsXhrError } from '../common/types/api';
 import type { FileActivity, FileActivityTypes } from '../common/types/feed';
 
 // We only show the latest reply in the UI
 const REPLY_LIMIT = 1;
 
-const getFileActivityQueryParams = (fileID: string, activityTypes?: FileActivityTypes[] = []) => {
+const getFileActivityQueryParams = (
+    fileID: string,
+    activityTypes?: FileActivityTypes[] = [],
+    shouldShowReplies?: boolean = false,
+) => {
     const baseEndpoint = `/file_activities?file_id=${fileID}`;
     const hasActivityTypes = !!activityTypes && !!activityTypes.length;
-    const enabledRepliesQueryParam = `&enable_replies=true&reply_limit=${REPLY_LIMIT}`;
+    const enableReplies = shouldShowReplies ? 'true' : 'false';
+    const enabledRepliesQueryParam = `&enable_replies=${enableReplies}&reply_limit=${REPLY_LIMIT}`;
```

Comment:
minor - you should be able to embed the boolean shouldShowReplies directly into the string and it will automatically convert to the string version of true or false (that way enableReplies const can be removed). Like:
```
`&enable_replies=${shouldShowReplies}&reply_limit=${REPLY_LIMIT}`
```

Reply:
linter has problems with not making it a string, kept as is